packages/pharos/src/components/toggle-button-group/PharosToggleButtonGroup.react.stories.mdx

```diff
@@ -0,0 +1,103 @@
+import { Story, Canvas, Meta, ArgsTable } from '@storybook/addon-docs/blocks';
+import { action } from '@storybook/addon-actions';
+
+import { PharosToggleButtonGroup } from '../../react-components/toggle-button-group/pharos-toggle-button-group';
+import { PharosToggleButton } from '../../react-components/toggle-button-group/pharos-toggle-button';
+
+<Meta title="Components/Toggle Button Group" />
```

Comment:
We haven't really settled on a name for this component -- if we decide to go with "button group," what would we call the button components themselves?